### DISCUSSION TITLE
Homematic CCU3  Geräte in HA einbinden

### DISCUSSION THREAD

**Discussion by ChristianHomeAssistant** [OpenCCU/OpenCCU#2828] [🇩🇪 Fragen&Antworten]
Hallo Leute,
bin blutiger Anfänger, schon mal sorry wenn ich komische Fragen stelle.

Gibt es eine Möglichkeit die Geräte und Variablen in den HA einzubinden, ohne RaspberryMatic auf der CCU zu installieren? So wie bei IO-Broker.

Danke im Vorraus
Christian

**Comment by Baxxy13** [comment]
Das Vorgehen mittels der empfohlenen Integration ist im Wiki beschrieben.
https://github.com/jens-maus/RaspberryMatic/wiki/HomeAssistant-Integration

Dabei spielt es keine Rolle welche Homematic-Zentrale du einsetzt. 
Ab CCU2 lässt sich alles integrieren. (nicht die Cloud-Lösung mit dem HAP)  

Du musst halt nur im 2.Teil der Anleitung bei Punkt: 8 die IP-Adresse deiner CCU eintragen.

**Reply by ChristianHomeAssistant** [reply]
Vielen dank für die Info
